start-test-server.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-atlas-run -u 6.3.0 --jvmargs "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005"
+atlas-run -u 6.3.14 --jvmargs "-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005"
```

Comment:
Just updated to match the AMPS version upgrade I applied in your `pom.xml`.

Reply:
Thanks!